tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_DuDt.cpp

```diff
@@ -0,0 +1,138 @@
+  CHECK(dt_psi.get(1, 1)[0] == approx(42.830161652334105));
+  CHECK(dt_psi.get(1, 2)[0] == approx(107.866588981051052));
+  CHECK(dt_psi.get(1, 3)[0] == approx(23.796407318352021));
+  CHECK(dt_psi.get(2, 2)[0] == approx(-37.597199305679588));
+  CHECK(dt_psi.get(2, 3)[0] == approx(-40.818346915888938));
+  CHECK(dt_psi.get(3, 3)[0] == approx(-81.239309564459717));
+  CHECK(dt_pi.get(0, 0)[0] == approx(-58401.962008002992661));
+  CHECK(dt_pi.get(0, 1)[0] == approx(-52279.223067088765674));
+  CHECK(dt_pi.get(0, 2)[0] == approx(-24296.935088025067671));
+  CHECK(dt_pi.get(0, 3)[0] == approx(15150.224762307567289));
+  CHECK(dt_pi.get(1, 1)[0] == approx(99233.234014284331352));
+  CHECK(dt_pi.get(1, 2)[0] == approx(121380.081659101357218));
+  CHECK(dt_pi.get(1, 3)[0] == approx(33311.102708022066508));
+  CHECK(dt_pi.get(2, 2)[0] == approx(68336.921757144518779));
+  CHECK(dt_pi.get(2, 3)[0] == approx(-39831.589825723327522));
+  CHECK(dt_pi.get(3, 3)[0] == approx(-120390.170747809068416));
+  CHECK(dt_phi.get(0, 0, 0)[0] == approx(9595.644222665809139));
+  CHECK(dt_phi.get(1, 0, 0)[0] == approx(-6060.591696572672845));
+  CHECK(dt_phi.get(2, 0, 0)[0] == approx(409.157616929634230));
+  CHECK(dt_phi.get(0, 0, 1)[0] == approx(1593.957300791164471));
+  CHECK(dt_phi.get(1, 0, 1)[0] == approx(688.907300097591587));
+  CHECK(dt_phi.get(2, 0, 1)[0] == approx(714.030346346540455));
+  CHECK(dt_phi.get(0, 0, 2)[0] == approx(6308.586720630850323));
+  CHECK(dt_phi.get(1, 0, 2)[0] == approx(-5262.127028145441727));
+  CHECK(dt_phi.get(2, 0, 2)[0] == approx(-4403.486993074658130));
+  CHECK(dt_phi.get(0, 0, 3)[0] == approx(1886.908426725741492));
+  CHECK(dt_phi.get(1, 0, 3)[0] == approx(-612.061820388978958));
+  CHECK(dt_phi.get(2, 0, 3)[0] == approx(-2401.251761757883287));
+  CHECK(dt_phi.get(0, 1, 1)[0] == approx(-2903.410866918160536));
+  CHECK(dt_phi.get(1, 1, 1)[0] == approx(561.848425485789903));
+  CHECK(dt_phi.get(2, 1, 1)[0] == approx(194.702313554982567));
+  CHECK(dt_phi.get(0, 1, 2)[0] == approx(-1354.731806920370673));
+  CHECK(dt_phi.get(1, 1, 2)[0] == approx(1275.757770912120350));
+  CHECK(dt_phi.get(2, 1, 2)[0] == approx(7132.000490989698847));
+  CHECK(dt_phi.get(0, 1, 3)[0] == approx(-7599.321000180396368));
+  CHECK(dt_phi.get(1, 1, 3)[0] == approx(5158.464428116095405));
+  CHECK(dt_phi.get(2, 1, 3)[0] == approx(6704.771543712529819));
+  CHECK(dt_phi.get(0, 2, 2)[0] == approx(-1594.948527344897002));
+  CHECK(dt_phi.get(1, 2, 2)[0] == approx(305.415616562497689));
+  CHECK(dt_phi.get(2, 2, 2)[0] == approx(3515.589817704739744));
+  CHECK(dt_phi.get(0, 2, 3)[0] == approx(-3238.223829450362246));
+  CHECK(dt_phi.get(1, 2, 3)[0] == approx(1001.253247998817869));
+  CHECK(dt_phi.get(2, 2, 3)[0] == approx(7182.548651343484380));
+  CHECK(dt_phi.get(0, 3, 3)[0] == approx(2563.754980467487712));
+  CHECK(dt_phi.get(1, 3, 3)[0] == approx(-2327.301264139446630));
+  CHECK(dt_phi.get(2, 3, 3)[0] == approx(-8207.442591423865451));
```

Comment:
(super fussy comment, feel free to ignore it:) Reorder these lines of code so that there is consistency in whether first or last index varies fastest.